homeassistant/components/tradfri/manifest.json

```diff
@@ -7,5 +7,5 @@
   "homekit": {
     "models": ["TRADFRI"]
   },
-  "codeowners": ["@ggravlingen"]
+  "codeowners": []
```

Comment:
Is it supposed to be an empty list or None?

Reply:
Empty list is ok.